tests/conftest.py

```diff
@@ -1120,3 +1120,14 @@ def mock_python_model_adapter_load_model_from_artifact():
 @pytest.fixture
 def endpoint_prediction_methods():
     return ["/predict/", "/predictions/", "/invocations"]
+
+
+@pytest.fixture
+def skip_if_running_inside_container():
+    skip = False
+    if os.path.exists("/.dockerenv"):
+        skip = True
+    if os.environ.get("KUBERNETES_SERVICE_HOST", None):
+        skip = True
+    if skip:
+        pytest.skip("Tests are running inside container")
```

Comment:
1. Please add the ticket number that will handle it, to the skip message (.e.g `RAPTOR-10673`). It will be easy later to locate all the related tests that should be enabled.
2. nit:
```suggestion
    skip = os.path.exists("/.dockerenv") or os.environ.get("KUBERNETES_SERVICE_HOST", None)
    if skip:
        pytest.skip("Tests are running inside container")
```